navit/support/gettext_intl/dcigettext.c

```diff
@@ -133,6 +133,11 @@ extern int errno;
 # define offsetof(type,ident) ((size_t)&(((type*)0)->ident))
 #endif
 
+#if defined(__MINGW32__) && !defined(__MINGW32CE__)
+# define HAVE_MEMPCPY   1
+#endif
+
+
```

Comment:
As the build in circleci now fails with
```
support/gettext_intl/libsupport_gettext_intl.a(dcigettext.c.obj): In function `libintl_dcigettext':
/root/project/navit/support/gettext_intl/dcigettext.c:575: undefined reference to `_mempcpy'
support/gettext_intl/libsupport_gettext_intl.a(localealias.c.obj): In function `read_alias_file':
/root/project/navit/support/gettext_intl/localealias.c:230: undefined reference to `_mempcpy'
collect2: ld returned 1 exit status
```
it looks as something is wrong with that condition now.

Reply:
@metalstrolch: The issue is that I included the changes by @bignaux during the development. Without them not very surprisingly, the build fails

Reply:
ah ok.

Reply:
Ok so far.